viewing_party/main.py

```diff
@@ -0,0 +1,108 @@
+def create_movie(title, genre, rating):
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie_dict = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+        
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    if len(user_data["watched"]) == 0:
+        return 0.0
+    else:
+        total = 0
+        for movie in user_data["watched"]:
+            total += movie["rating"]
+        average_rating = total / len(user_data["watched"])
+    return average_rating
+
+def get_most_watched_genre(user_data):
+    if len(user_data["watched"]) == 0:
+        return None
+
+    popular_genre = {}
+    for movie in user_data["watched"]:
+        current_genre = movie["genre"]
+        popular_genre[current_genre] = popular_genre.get(current_genre, 0) + 1
```

Comment:
Wow, very succinct! Awesome! I'm curious to hear how you learned about this `get` method!